docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '2'
+services:
+  app: &base
+    build: .
+    working_dir: /code
+    command: bash -c 'composer install -n && set -a && composer ci'
```

Comment:
co dělá `set -a`? A proč se volá composer install - ten je součástní dockerimage, ne? Dtto working dir - ten je to kde skončí dockerfile... ne?

Reply:
smazal jsem celý command